try {
  console.log('linea de try 1');
  variableFantasma;
  console.log('linea de try 2');
} catch (error) {
  console.log('esto es un error');
}